components/authz-service/engine/opa/opa_internal_test.go

```diff
@@ -549,3 +429,866 @@ func randomPolicies(i int) []map[string]interface{} {
 
 	return newPolicies
 }
+
+// v2 benchmarks
+
+var (
+	allProjects = []string{
+		"(unassigned)",
+		"project1",
+		"project2",
+		"project3",
+		"project4",
+		"project5",
+		"project6",
+	}
+	projectsResponse []string
```

Comment:
the go compiler will eliminate the result of the benchmark if you don't save it as a package variable. ([fun fact from this helpful post](https://dave.cheney.net/2013/06/30/how-to-write-benchmarks-in-go)) `projectsResponse` is the response type for `FilterAuthorizedProjects` and `ProjectsAuthorized`